packages/test/test-end-to-end-tests/src/test/gc/newFluidObjectVisibility.ts

```diff
@@ -0,0 +1,361 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import assert from "assert";
+import { IContainer } from "@fluidframework/container-definitions";
+import { Container } from "@fluidframework/container-loader";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
+import { IFluidHandle, IFluidRouter, IRequest } from "@fluidframework/core-interfaces";
+import { SharedMap } from "@fluidframework/map";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+import { ITestObjectProvider } from "@fluidframework/test-utils";
+import {
+    describeNoCompat,
+    ITestDataObject,
+    TestDataObjectType,
+} from "@fluidframework/test-version-utils";
+
+/**
+ * These tests validate that new Fluid objects such as data stores and DDSs become visible correctly. For example,
+ * new non-root data stores should not become visible (or reachable from root) until their handles are added to a
+ * visible DDS.
+ */
+describeNoCompat.only("New Fluid objects visibility", (getTestObjectProvider) => {
```

Comment:
remove .only

Reply:
this file would be easier to read if all the functions, and the reused tests were not nested in the describe